backend/src/indexer/block_preprocessor.rs

```diff
@@ -194,22 +196,27 @@ impl concordium_rust_sdk::indexer::Indexer for BlockPreProcessor {
 
             let get_tokenomics_info = async move {
                 let tokenomics_info = client3.get_tokenomics_info(fbi.height).await?.response;
-                let total_staked_capital = match &tokenomics_info {
+                let computed_validator_staking_details = match &tokenomics_info {
                     RewardsOverview::V0 {
                         ..
                     } => {
-                        compute_total_stake_capital(
+                        compute_validator_staking_information(
                             &mut client3,
                             v2::BlockIdentifier::AbsoluteHeight(fbi.height),
                         )
                         .await?
                     }
                     RewardsOverview::V1 {
-                        total_staked_capital,
                         ..
-                    } => *total_staked_capital,
+                    } => {
+                        compute_validator_staking_information(
+                            &mut client3,
+                            v2::BlockIdentifier::AbsoluteHeight(fbi.height),
+                        )
+                        .await?
+                    }
```

Comment:
since we don't do any difference between the `rewardsOverview::V0` and `rewardsOverview::V1` arms, I don't think we need the match statement anymore.

```
       compute_validator_staking_information(
                            &mut client3,
                            v2::BlockIdentifier::AbsoluteHeight(fbi.height),
                        )
                        .await?
```

Reply:
Question is do we then need the query to `tokenomics_info` endpoint? Might still be needed.